crates/cxx-qt-gen/src/generator/cpp/signal.rs

```diff
@@ -103,8 +102,7 @@ pub fn generate_cpp_signal(
     let free_connect_ident_cpp = idents_helper.connect_name.cpp;
 
     // Retrieve the parameters for the signal
-    let parameters =
-        parameter_types_and_values(&signal.parameters, type_names, &qobject_ident_namespaced)?;
+    let parameters = parameter_types_and_values(&signal.parameters, type_names, &qobject_ident)?;
```

Comment:
why is this not namespaced when it was before?

Reply:
I honestly have no idea why it was before.
The `parameter_types_and_values` will namespace it using `cxx_qualified`.
It think before it only worked because cxx_qualified does a passthrough.